src/test/java/com/splunk/kafka/connect/SplunkSinkConnectorConfigTest.java

```diff
@@ -238,6 +238,17 @@ public void createWithMetaDataError() {
         SplunkSinkConnectorConfig connectorConfig = new SplunkSinkConnectorConfig(config);
     }
 
+    @Test
```

Comment:
Add more test cases, 
1. Using topics and topics.regex, maybe we should throw an error?
2. Using topics.regex
3. Using topics

Reply:
1) when we define topics and topics.regex in connector task config then the below error comes from Kafka so we do not need to throw an error. and connector task is not created.
[org.apache.kafka.common.config.ConfigException: topics and topics.regex are mutually exclusive options]
3) There are sufficient testcases available in SplunkSinkConnectorConfigTest class related to topics functionality